test/check-ostree

```diff
@@ -440,24 +440,29 @@ class OstreeRestartCase(testlib.MachineCase):
 
         # start our local registry
         self.start_oci_registry()
+        oci_repo = "ostree-unverified-registry:localhost:5000/ostree-oci"
+        oci_branch = "cockpit1"
+        oci_repo_branch = f"{oci_repo}:{oci_branch}"
 
         self.login_and_go("/updates")
 
         # our image defaults to local OSTree repo
+        b.wait_text("#current-repository .pf-v5-c-description-list__description", "local")
+        b.wait_text("#current-branch .pf-v5-c-description-list__description", branch)
         wait_deployment_prop(b, 1, "Version", "cockpit-base.1")
         wait_deployment_prop(b, 1, "Status", "Current")
         wait_deployment_details_prop(b, 1, "Tree", ".origin", f"local:{branch}")
         wait_packages(b, 1, {"rpms-col1": [bash_ver]})
 
         # rebase to our OCI repo
         # FIXME: there is currently no UI way of doing this
```

Comment:
Side-quest, I recall a rebase option in the UI these days. Not for this PR but maybe interesting as follow up?

Reply:
Yup, working towards that, see https://issues.redhat.com/browse/COCKPIT-928